### ISSUE TITLE
Repo name is a bit misleading for iotedge scenario

### ISSUE THREAD

**Issue by jiata** [Microsoft/MLAKSDeployAML#33]

**Comment by msalvaris** [comment]
@danielleodean I think we should change the name, Thoughts ?

**Comment by danielleodean** [comment]
yeah it's good point - are we going to separate the IOT stuff into a separate repo though? (still using same approach but full notebooks in each?) if we're doing that in short term, maybe just keep the name, since it can go back to just being about AKS?

**Comment by msalvaris** [comment]
@danielleodean I was more thinking of keeping the deployments together but maybe it make sense to seperate them. They can be based off the same CC since the overal is considerable. Thoughts?

**Comment by danielleodean** [comment]
Yeah was thinking same CC but maybe different repos? as there are considerations like repo name, different architecture diagrams, different performance considerations that might want to have separate readmes also? But if there are too many downsides to this, I'm good with whatever option you all decide @YanZhangADS also fyi

**Comment by msalvaris** [comment]
Yep, that works for me :) @YanZhangADS what do you think?
